composer.json

```diff
@@ -6,13 +6,16 @@
     "license": "MIT",
     "minimum-stability": "dev",
     "require": {
-        "php": "^7.2",
-        "itk-dev/pretix-api-client-php": "^1.0",
+	    "php": "^8.1",
```

Comment:
This line is not indented like other similar lines in this file.

Reply:
Fixed